include/mvdtool/bits/tsv_misc.hpp

```diff
@@ -71,11 +71,17 @@ inline TSVFile::unordered_pair_map readTSVFile(const std::string& filename,
         ensure_correct_n_fields(line_info.size());
 
         auto morph_name = line_info[MEComboEntry::MorphologyName];
+        int32_t layer = 0;
```

Comment:
Following Jim's suggestion, I think we should allow the user to read whatever this filed contains. The TSV reading support here is a recent thing and has not been used AFAIK, so we don't risk much changing the API.
As a suggestion, instead std::string, this could be e.g. char[8]

Reply:
Why limit the layer field to `*char`, though?

Reply:
> Why limit the layer field to *char, though?

should it be a union? 😆

I feels a waste to spend 24 or 32 bytes for something that is typically 1 byte, max 5. 
At least recent std::string's implement the small buffer optimization to avoid touching the heap, so if it's easier to implement with `std:string` I have nothing against.
(some ref: https://shaharmike.com/cpp/std-string/.)

Reply:
Hello @ferdonline ,

I have tried to change the tsv layer type to ```string``` and it broke the function ```MVD3File::getLayers()```.  I've seen that it reads the layer info from either tsv or mvd3 files. In mvd3 file the ```/cells/properties/layer`` data type is H5T_STD_I32LE (int).

```
inline std::vector<int32_t> MVD3File::getLayers(const Range& range) const {
    return getDataFromTSVorMVD<int32_t>(did_cells_layer, did_lib_NONE, TSVColumn::Layer, range);
}
```
```
template <typename T>
inline std::vector<T> MVD3File::getDataFromTSVorMVD(const std::string& did_ds,
                                                    const std::string& did_lib,
                                                    const TSVColumn& col,
                                                    const Range& range) const {
    if (_tsv_file) {
        return getDataFromTSV<T>(col, range);
    }
    return getDataFromMVD<T>(did_ds, did_lib, range);
}
```